scripts/versions.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
 # Don't export them as they're used in the context of other calls
-avalanche_version=${AVALANCHE_VERSION:-'v1.9.16'}
+avalanche_version=${AVALANCHE_VERSION:-'v1.10.2'}
```

Comment:
I wonder if we can fetch this from go.mod instead of hardcoding here.  Not necessarily to be in scope of this PR.